package.json

```diff
@@ -8,7 +8,7 @@
   "browserslist": "last 2 versions",
   "dependencies": {
     "@lhci/cli": "^0.7.0",
-    "@saleor/sdk": "^0.3.0-8",
+    "@saleor/sdk": "git://github.com/mirumee/saleor-sdk.git#attributes_update",
```

Comment:
I think it should be pinned to a commit, so we we can be sure dependencies are stable. Or do you plan to release sdk before merging this pr?

Reply:
@dominik-zeglen Yes, waiting for release.